library/core/src/ops/bit.rs

```diff
@@ -207,7 +207,10 @@ bitand_impl! { bool usize u8 u16 u32 u64 u128 isize i8 i16 i32 i64 i128 }
 ///     fn bitor(self, Self(rhs): Self) -> Self::Output {
 ///         let Self(lhs) = self;
 ///         assert_eq!(lhs.len(), rhs.len());
-///         Self(lhs.iter().zip(rhs.iter()).map(|(x, y)| *x || *y).collect())
+///         Self(lhs.iter()
+///                 .zip(rhs.iter())
+///                 .map(|(x, y)| *x | *y)
+///                 .collect())
```

Comment:
`rustfmt` doesn't automatically format doc examples, but it's probably best to just keep `rustfmt`'s style here:
```suggestion
///         Self(lhs.iter().zip(rhs.iter()).map(|(x, y)| *x | *y).collect())
```
---

Looks like the other three examples already didn't follow `rustfmt`s style. (It does spread those over multiple lines, but a bit differently.) Feel free to fix those too, but that doesn't have to be part of this PR.